int16 can_id
int16 rpm
int16 RMS_current
int16 DC_voltage
int16 drive6stat
